bugalink-frontend/components/ProfileHeader/index.tsx

```diff
@@ -14,6 +17,7 @@ export default function ProfileHeader({
   numberOfRatings,
   className = '',
   photo,
+  id = 1,
```

Comment:
Should not default to id=1, imagine there is no ID and you get redirected to a random user's profile instead